CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Welcome to the Contribution Guide for the MACH Interoperability Standards Repository
+
+Thanks for your interest in contributing to the MACH Interoperability Standards repository. Before you dive in, please check out these guidelines. Not following them may result in your Pull Request/issue being closed.
+
+## Code of Conduct
+
+We follow the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/version/1/4/code-of-conduct/), which is widely accepted as a standard in communities. Let's keep it respectful and kind.
```

Comment:
The linked page has a placeholder for [INSERT EMAIL ADDRESS]. Should we write here what is the email address for private reporting of potential CoC violations?

Reply:
Let's discuss it on today's interoperability standards group. For now I only added @adampnielsen GH handle as potential contact person